src/main/java/org/octri/hpoOnFhir/HpoOnFhirApplication.java

```diff
@@ -2,8 +2,12 @@
 
```

Comment:
One comment. the typical Java convention is for lowercase package names. It's only a convention, but I thought I'd mention it.

Reply:
Thanks. I'll make that change.